h2o-docs/src/product/getting-data-into-h2o.rst

```diff
@@ -190,52 +192,135 @@ core-site.xml must be configured for at least the following properties (class, p
     </configuration>
 
 
-Direct Hive import
+Direct Hive Import
 ~~~~~~~~~~~~~~~~~~
 
-H2O supports direct ingestion of data managed by Hive in Hadoop. This feature is available only when H2O is running as a Hadoop job. Internally H2O uses metadata in Hive Metastore database to determine the location and format of given Hive table. H2O then imports data directly from HDFS so limitations of supported formats mentioned above apply. Data from hive can pulled into H2O using ``import_hive_table`` function. H2O can read Hive table metadata two ways - either via direct Metastore access or via JDBC.
+H2O supports direct ingestion of data managed by Hive in Hadoop. This feature is available only when H2O is running as a Hadoop job. Internally H2O uses metadata in Hive Metastore database to determine the location and format of given Hive table. H2O then imports data directly from HDFS so limitations of supported formats mentioned above apply. Data from hive can pulled into H2O using ``import_hive_table`` function. H2O can read Hive table metadata two ways - either via direct Metastore access (preferred) or via JDBC. 
 
-Requirements:
+Requirements
+''''''''''''
 
-- Direct Metastore access - Hive jars and configuration must be present on H2O job classpath - either via adding it to yarn.application.classpath (or similar property for your resource manger of choice) or by adding Hive jars and configuration to libjars.
-- JDBC metadata access - Hive JDBC Driver must be on H2O job classpath. (See below for details about access Kerberized Hive via JDBC.)
 - The user running H2O must have read access to Hive and the files it manages.
 
+- For Direct Metastore access, the Hive jars and configuration must be present on H2O job classpath - either by adding it to yarn.application.classpath (or similar property for your resource manger of choice) or by adding Hive jars and configuration to libjars. Note that Direct Metastore access is the perferred method.
```

Comment:
same as above

Reply:
So now I'm confused about the Jira ticket: https://0xdata.atlassian.net/browse/PUBDEV-7343
> Some people end-up using JDBC interface which is known to be sub-optimal.
> We need to make sure there is a clearly preferred method, with a clear description...

Reply:
the ticket is about using JDBC http://docs.h2o.ai/h2o/latest-stable/h2o-docs/getting-data-into-h2o.html#jdbc-databases vs using filesystem or direct hive import http://docs.h2o.ai/h2o/latest-stable/h2o-docs/getting-data-into-h2o.html#direct-hive-import